src/decoderWorker.js

```diff
@@ -30,10 +30,31 @@ var OggOpusDecoder = function( config ){
   this.decoderSampleRate = config['decoderSampleRate'] || 48000;
   this.outputBufferSampleRate = config['outputBufferSampleRate'] || 48000;
   this.resampleQuality = config['resampleQuality'] || 3;
+  this.rawPacket = config['rawPacket'] || false;
   this.outputBuffers = [];
+  if (this.rawPacket){ this.inited = false };
 };
 
 OggOpusDecoder.prototype.decode = function( typedArray ) {
+  if (this.rawPacket) {
```

Comment:
I'm a fan of one return point for each method. IMO a code block of this size should be broken into two. Perhaps `OggOpusDecoder.prototype.decodeRawPacket` and `OggOpusDecoder.prototype.decodeOggContainer` or something like that.